front/cypress/e2e/admin/pages_and_menu/update_hero_banner_content.cy.ts

```diff
@@ -64,6 +95,9 @@ describe('Admin: update Hero Banner content', () => {
   });
 
   it('updates hero banner settings as admin correctly', () => {
+    cy.intercept('PATCH', '**/home_page').as('saveHomePage');
```

Comment:
Nice! One more way to avoid `sleep`.

Reply:
Shout out to @IvaKop for introducing me to this, it's super useful (to avoid `wait`) but you can also do a lot of interesting things with the requests you intercept, checking statuses and values and such